Git: http://git.kernel.org/?p=linux/kernel/git/sameo/mfd-2.6.git;a=commitdiff;h=45e48aa6cabab6b03ac62caded4d86f157997b13


From: Samuel Ortiz <sameo@linux.intel.com>
Date: Thu, 8 Nov 2012 09:38:56 +0000 (+0100)
Subject: mfd: Select MFD_CORE for rtsx
X-Git-Tag: mfd-3.8-1~107
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fsameo%2Fmfd-2.6.git;a=commitdiff_plain;h=45e48aa6cabab6b03ac62caded4d86f157997b13

mfd: Select MFD_CORE for rtsx

The realtek driver use the MFD core API and thus must select MFD_CORE.

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
---

diff --git a/drivers/mfd/Kconfig b/drivers/mfd/Kconfig
index d0cb4d4..3f2187a 100644
--- a/drivers/mfd/Kconfig
+++ b/drivers/mfd/Kconfig
@@ -66,6 +66,7 @@ config MFD_SM501_GPIO
 config MFD_RTSX_PCI
 	tristate "Support for Realtek PCI-E card reader"
 	depends on PCI
+	select MFD_CORE
 	help
 	  This supports for Realtek PCI-Express card reader including rts5209,
 	  rts5229, rtl8411, etc. Realtek card reader supports access to many
